### ISSUE TITLE
Multi-line arrays require extra indentation to be parsed correctly

### ISSUE THREAD

**Issue by jpetazzo** [aws/amazon-ecs-cli#12]
When I try the following Compose file with ECS CLI:
```
redis:
  image: redis
  ports:
  - 10000:6379
```

I get the following error:
```
ERRO[0000] Unable to open ECS Compose Project            error=Unable to unmarshal compose config. Error: interface conversion: yaml.Node is yaml.Map, not yaml.List
FATA[0000] Unable to create and read ECS Compose Project  error=Unable to unmarshal compose config. Error: interface conversion: yaml.Node is yaml.Map, not yaml.List
```

Adding two spaces in front of the `-` fixes the issue. The original Compose file is valid YAML, though.

**Comment by sporokh** [comment]
Any fixes for this issue?
Got the same error:
```
DEBU[0000] Parsing the compose yaml...                  
ERRO[0000] Unable to open ECS Compose Project            error=Unable to unmarshal compose config. Error: interface conversion: yaml.Node is yaml.Map, not yaml.List
FATA[0000] Unable to create and read ECS Compose Project  error=Unable to unmarshal compose config. Error: interface conversion: yaml.Node is yaml.Map, not yaml.List
```

**Comment by christianbradley** [comment]
Oy? Anything on this?

**Comment by uttarasridhar** [comment]
fixed in #121 and released in [v0.4.0](https://github.com/aws/amazon-ecs-cli/releases/tag/v0.4.0).
